src/azure-cli/azure/cli/command_modules/appservice/resources/LinuxFunctionsStacks.json

```diff
@@ -216,9 +216,9 @@
                             "use32BitWorkerProcess": false,
                             "linuxFxVersion": "Python|3.9"
                         },
-                        "isPreview": false,
+                        "isPreview": true,
                         "isDeprecated": false,
-                        "isHidden": false
```

Comment:
Once changes to support python 3.9 are deployed to all regions in Azure, you need to send another PR to un-hide

Reply:
in other words - if changes are already deployed to Azure, no need to set the `isHidden` property - just setting preview to true is enough